### DISCUSSION TITLE
License Key for  each quantity of same product

### DISCUSSION THREAD

**Discussion by archanakv893** [duncanmcclean/simple-commerce#1189] [Feature Requests]
I want to add logic that creates a license key for each quantity of the same product. For example, if a customer places an order for digital product 1 with quantity 3, then the system should generate 3 license keys instead of 1.

**Comment by archanakv893** [comment]
Please any one help me out with this

**Comment by duncanmcclean** [comment]
You'll either need to ensure that [each quantity is it's own "line item"](https://simple-commerce.duncanmcclean.com/tags/cart#content-adding-the-same-product-multiple-times) on the order, or you'll need to handle the issuing of license keys yourself.

As a heads up, license keys for digital downloads are being removed in the next major version as it's barely used.
